Makefile

```diff
@@ -34,37 +34,40 @@ mocks:
 	@mockgen -destination ${MOCK_DIR}/mock_store/mocks.go github.com/GoogleCloudPlatform/docker-credential-gcr/store GCRCredStore
 	@mockgen -destination ${MOCK_DIR}/mock_config/mocks.go github.com/GoogleCloudPlatform/docker-credential-gcr/config UserConfig
 	@mockgen -destination ${MOCK_DIR}/mock_cmd/mocks.go github.com/GoogleCloudPlatform/docker-credential-gcr/util/cmd Command
-	
+
 strip-vendor-dependencies:
-	@sed -i -e 's/github.com\/GoogleCloudPlatform\/docker-credential-gcr\/vendor\///g' ${SRCS}
+# The differences in -i's behavior on OSX and linux necessitate the creation 
+# of .bak files, which we want to clean up afterward... 
+	@sed -i.bak -e 's/github.com\/GoogleCloudPlatform\/docker-credential-gcr\/vendor\///g' ${SRCS}
```

Comment:
You can use any character to delimit things in `sed`, so instead of escaping the slashes delimit things differently, e.g. `sed -i 's@my/path@new/path@g'`.

This `sed` line altogether points at some deeper issue, and likely a much longer discussion.

Reply:
roger, implemented it before I knew about that. The sed is a hack to fix erroneous import paths generated by gomock:
https://github.com/golang/mock/issues/30
I think the fix will to be to limit the search to sources in the `mock` directory